CryptoTechReminderSystem.Test/Gateway/SlackGatewayTests.cs

```diff
@@ -99,19 +145,58 @@ public void CanSendAPostMessageRequestWithAChannelAndText()
             {
                 var channel = "U98ZL999";
                 var text = "Please make sure your timesheet is submitted by 13:30 on Friday.";
-                var message = new Message()
+                var message = new Message
                 {
                     Channel = channel,
                     Text = text
                 };
             
                 _slackGateway.Send(message);
             
-                var receivedRequest = _slackApi.ReceivedRequests.First();
+                var receivedRequest = _slackApi.ReceivedRequests().First();
                 
                 JObject.Parse(receivedRequest.RequestBody)["channel"].ToString().Should().Be(channel);
                 JObject.Parse(receivedRequest.RequestBody)["text"].ToString().Should().Be(text);
             }
+            
+            [Test]
+            public void CanSendAPostMessageRequestWithSuccess()
+            {
+                var channel = "U98ZL999";
+                var text = "Please make sure your timesheet is submitted by 13:30 on Friday.";
+                var message = new Message
+                {
+                    Channel = channel,
+                    Text = text
+                };
+                
+                var response = _slackGateway.Send(message);
+
+                response.OnSuccess(f => _slackApi.SpyMethod());
```

Comment:
This `f` definitely needs to be renamed.

Reply:
Renamed in https://github.com/chubberlisk/crypto_tech/pull/37/commits/94b030d290d4a33474c393f25c1ae7f8f04fa51b